src/libs/web3-data-provider/provider.tsx

```diff
@@ -174,26 +174,31 @@ export function Web3Provider({
       setCurrentProviderName(connectorName);
       isSuccessful = true;
     } catch (e) {
+      console.log('eee', e.message);
```

Comment:
The logging can be removed. I'd suggest remove logging before request code review.